src/arguments.js

```diff
@@ -61,10 +62,15 @@ const getDefaultValue = (description, regexp) => {
  * @return {array} - enum values of argument
  */
 const getEnum = (string, context) => {
-    const result = getValueByRegexp(string, context.regexp.enumValues.enum);
-    return result ? removeExtraEnumValues(
-        result[2].split(context.regexp.enumValues.split), context.regexp.path)
-        : null;
+    let enumValues = getValueByRegexp(string, context.regexp.enumValues.enum);
+    if (enumValues) {
+        enumValues = enumValues[2].split(context.regexp.enumValues.split);
+        enumValues = enumValues.map((res) =>
+            removeChar(res, ['<', '>', '\'', '\"']).trim());
+        enumValues = removeExtraEnumValues(enumValues, context);
```

Comment:
there is no need to save result of `removeExtraEnumValues`. We can return it at once